nmrfx-processor-gui/src/main/java/org/nmrfx/processor/gui/FXMLController.java

```diff
@@ -112,47 +112,27 @@ public class FXMLController implements  Initializable, PeakNavigable {
     private StackPane chartPane;
     @FXML
     private VBox phaserBox;
-    @FXML
-    private VBox vectorBox;
-
     @FXML
     private BorderPane borderPane;
     @FXML
     private BorderPane mainBox;
     @FXML
     private StackPane processorPane;
     @FXML
-    private HBox dimHBox;
-    @FXML
-    private HBox dimHBox2;
-    @FXML
-    private Slider vecNum1;
-    @FXML
     private GridPane rightBox;
-    private TextField[] rowTextBoxes = new TextField[0];
-    private TextField fileIndexTextBox = new TextField();
-    ToggleGroup rowToggleGroup = new ToggleGroup();
-    private ChoiceBox<String> realImagChoiceBox = new ChoiceBox();
-    private List<String> realImagChoices = new ArrayList<>();
-    ChangeListener<String> vecNumListener;
-    int[] rowIndices;
-    int[] vecSizes;
-
     private Button cancelButton;
-    EventHandler<ActionEvent> menuHandler;
     PopOver popOver = null;
     PopOver attributesPopOver = null;
 
-    ChangeListener<String> dimListener;
-    ChangeListener<Number> refDimListener;
-
     ChartProcessor chartProcessor;
     DocWindowController dwc = null;
     static SpecAttrWindowController specAttrWindowController = null;
     static boolean popOverMode = false;
     static PeakAttrController peakAttrController = null;
-    ProcessorController processorController = null;
     Stage stage = null;
+    private double previousStageRestoreWidth = 0;
+    private double previousStageRestoreProcControllerWidth = 0;
+    private boolean previousStageRestoreProcControllerVisible = false;
```

Comment:
Allowing multiple FID processor controllers required setting the size in a different way, because the previous sizing method would continuously increase the stage size each time a new FID was loaded. The size is now adjusted in two methods: 

- adjustSizeAfterMaximize: Uses these three variables to adjust the size when restoring down after the window has been maximized. I don't really like this solution but I couldn't come up with anything else...any suggestions would be welcome.
- updateStageSize : Adjusts the size of the window when loading an FID or dataset